### ISSUE TITLE
Wine for OSX doesn't register Info.plist

### ISSUE THREAD

**Issue by gbonk** [Homebrew/homebrew-core#2465]
# Please follow the general troubleshooting steps first:

- [ yes] Ran `brew update` and retried your prior step?
- [ yes] Ran `brew doctor`, fixed as many issues as possible and retried your prior step?
- [ n/a] If you're seeing permission errors tried running `sudo chown -R $(whoami) $(brew --prefix)`?

_You can erase any parts of this template not applicable to your Issue._

### Bug reports:

Using Brew to install Wine does not register the Info.plist for MacOSX

Trying to hunt down the cause of https://bugs.winehq.org/show_bug.cgi?id=37047

Wine seems to think their side is fine.

There are no errors for "brew gist-logs wine"

### Formula Requests:

**Please note by far the quickest way to get a new formula into Homebrew is to file a [Pull Request](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md).**

We will consider your request but it may be closed if it's something we're not actively planning to work on.

**Comment by ilovezfs** [comment]
Yup, issue confirmed. Tarball does have it, but neither the bottle nor a source build via brew will install `wine_info.plist`.

**Comment by gbonk** [comment]
Awesome.  How long till fixed ???  :-)  It's killing me :-)

**Comment by KenThomases** [comment]
wine_info.plist is not supposed to be "installed" anywhere. It's incorporated into the wineloader executable when that is built. Running the command `otool -V -s __TEXT __info_plist <path to file>` will show it.

**Comment by ilovezfs** [comment]
Yeah, I just noticed this as well. So I'm not sure what it means to "register the Info.plist"

**Comment by ilovezfs** [comment]
Closing per upstream comment: https://bugs.winehq.org/show_bug.cgi?id=37047#c9

**Comment by gbonk** [comment]
Why not enhance the brew install to register it into the Launch Services Database?  Brew is the recommended way to install but only the PKG distro actually does that.

**Comment by ilovezfs** [comment]
As soon as upstream starts doing that in `make install` we can discuss it.

**Comment by gbonk** [comment]
Spent some time debugging this.  I manually installed wine with the 1.9.13 Tarball from 

https://dl.winehq.org/wine/source/1.9/wine-1.9.13.tar.bz2

and built it. ran it and when executing bin/wine the  Application is registered correctly and is recognized by Karabiner.

So there is NO issue with WINE.

Then I made a local copy of Homebrew/homebrew-core locally and used the devel to install wine

`brew install wine.rb --devel `

I ran wine locally...

`/usr/local/Cellar/wine/1.9.13/bin/wine.bin ~/.wine/drive_c/Program\ Files/Notepad++/notepad++.exe 
`

and then Karabiner does not properly recognise wine.

Looking at the Formula, I removed the wine_wrapper task from my local copy of the wine.rb formula.

Now when I ran and installed brew via local `brew install wine.rb --devel ` and then ran wine and an application, ( as above )

now Karabiner recognizes the wine application that is running.

So I think that is pretty clear that there is an issue in the wine.rb formula.  

Any other thoughts?  I'm not clear as to what the purpose of the wine_wrapper is for.  Wine is running great on my mac without it.
